src/test/java/mpack/Assignement4.java

```diff
@@ -0,0 +1,95 @@
+//Dynamic Xpath
+package mpack;
+
+import java.util.List;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.safari.SafariDriver;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+
+public class Assignement4 {
```

Comment:
Adding this for your information 
Please correct it 


Class Name
Convention: Use PascalCase (also known as UpperCamelCase).

Rules:

The first letter of each word in the class name should be capitalized.
The class name should be a noun or noun phrase that accurately describes the entity the class represents.
Avoid using acronyms, but if necessary, write them in uppercase (e.g., HTTPServer, XMLParser).
The class name should be descriptive but not too long.
Examples:

Employee
AccountManager
DatabaseConnection
UserProfile

Reply:
changed the class names